polyphemus/spec/metis_file_etl_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe Polyphemus::MetisFileEtl do
+  class TestEtl < Polyphemus::MetisFileEtl
+    def initialize(**args)
+      super(project_bucket_pairs: [['ipi', 'data'], ['ipi', 'waiver_data']], **args)
+      @cursor_group.cursors.each { |c| c[:limit] = 2} # Force paging on everything
+    end
+
+    attr_reader :process_calls
+    def process(cursor, files)
+      @process_calls ||= []
+      @process_calls << [cursor.updated_at.dup, cursor.value.dup, files.dup]
+    end
+  end
+
+  # To re-record this test, add RERECORD=1 and METIS_TOKEN=xxxxx to your environment before running.
+  it 'should run end to end' do
+    VCR.use_cassette('metis_file_etl.e2e') do
+      metis_client = Etna::Clients::Metis.new(host: 'https://metis.development.local', token: ENV['METIS_TOKEN'])
```

Comment:
VCR allows me to run a local test against live servers, record the interactions to a file, and then use that file for future tests (which don't require the live servers anymore).  Allows very simple to setup integration testing.

Reply:
Will you be committing the cassettes to the repo?

Reply:
Yes, I will.

Reply:
While it sounds dumb, we should guard against the possibility of PHI ending up in these cassettes, which might happen if we are pulling live data. Not sure how to do this other than vigilance.

Reply:
Should we set this test to a not-real-bucket but a test project and test buckets, as part of that? We would all have to synch on the structure and contents of that...

Reply:
In this case, interestingly enough, no file contents are actually pulled -- only the file metadata (path, download url, updated_at).  Check the cassette file to double check.

In general I agree that we should create some tools to create useful test data to reduce this risk.  This test, as written, can actually be run against any source of data anyways, it's expectations aren't particularly tied to any dataset.

Reply:
What about potentially encrypting the cassettes with a secret?  CI and local users can decrypt using some common secret, but the committed file is opaque.

Reply:
Would that process be automated for whenever we re-record the cassette? And automated decryption for local testing?